src/Scenario/Processor.php

```diff
@@ -0,0 +1,82 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\MagentoCloud\Scenario;
+
+use Magento\MagentoCloud\App\GenericException;
+use Magento\MagentoCloud\Package\Manager;
+use Magento\MagentoCloud\Process\ProcessInterface;
+use Magento\MagentoCloud\Scenario\Exception\ProcessorException;
+use Psr\Log\LoggerInterface;
+
+/**
+ * Process given scenarios.
+ */
+class Processor
+{
+    /**
+     * @var Merger
+     */
+    private $merger;
+
+    /**
+     * @var LoggerInterface
+     */
+    private $logger;
+
+    /**
+     * @var Manager
+     */
+    private $manager;
+
+    /**
+     * @param Merger $merger
+     * @param LoggerInterface $logger
+     * @param Manager $manager
+     */
+    public function __construct(Merger $merger, LoggerInterface $logger, Manager $manager)
+    {
+        $this->merger = $merger;
+        $this->logger = $logger;
+        $this->manager = $manager;
+    }
+
+    /**
+     * @param array $scenarios
+     * @throws ProcessorException
+     */
+    public function execute(array $scenarios)
+    {
+        $this->logger->info(sprintf(
+            'Starting scenario(s): %s %s',
```

Comment:
What is the example of this message?
'Starting scenario(s): scenario/deploy.xml magento/ece-tools version: 2002.1.0, magento/magento2-base version: 2.3.3' ?
if so, I would suggest to change it

Reply:
Change it to what?

Reply:
It's currently `Starting scenario(s): scenario/build/generate.xml (magento/ece-tools version: dev-MAGECLOUD-4004, magento/magento2-base version: 2.3.2) `

Reply:
ok. It looks better then.